### PR TITLE
Added Description about the project 

### PR DESCRIPTION
**Pull request by Satarupa22-SD**
## Pull Request description

I have noticed that a project description was missing in the readme file. This would enable the project to be more accessible and increase its usability.

 improving documentation 

## How to test these changes

* ```...```

## Pull Request checklists

This PR is a:
- [ ] bug-fix
- [ ] new feature
- [x] maintenance

About this PR:
- [ ] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [ ] pre-commit hooks were executed locally.
- [x] this PR requires a project documentation update.

Author's checklist:
- [x] I have reviewed the changes and it contains no misspelling.
- [x] The code is well commented, especially in the parts that contain more complexity.
- [x] New and old tests passed locally.

## Reviewer's checklist

Copy and paste this template for your review's note:

```
## Reviewer's Checklist

- [ ] I managed to reproduce the problem locally from the `main` branch
- [ ] I managed to test the new changes locally
- [ ] I confirm that the issues mentioned were fixed/resolved .
```

### CODE REVIEW

**Review by xmnlab — CHANGES_REQUESTED** [review]
thanks for working on that @Satarupa22-SD !
In general looks good. could you add that information about that we aim to support the recommendations from pyopensci pls?

**Comment by xmnlab** [comment]
@Satarupa22-SD you maybe can use this as reference (feel free to change it):

> This cookiecutter Python template  aims to provide the workflow, libraries, and tools recommended by PyOpenSci. In addition, this template includes extra options that can improve the development workflow.
> PyOpenSci is currently conducting an investigation to identify tools, libraries, best practices, and workflows used by the most notable scientific Python communities.

**Comment by Satarupa22-SD** [comment]
@xmnlab added the details

File: README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Cookiecutter Python Package
 
+**Cookiecutter-python** is a template developed by [Open Science Labs](https://opensciencelabs.org/) that creates projects from project templates and is based on [Cookiecutter](https://github.com/cookiecutter/cookiecutter) command-line utility  . It serves as a boilerplate which can be used by beginners as well as full fledged developers to simplify the project creation process and save considerable amount of time.
+Cookiecutter enables projects with an initial layout that includes recommended tools, workflows, and project structure. 
+
+PyOpenSci is actively conducting research to determine the tools, libraries, best practices, and workflows utilized by significant scientific Python groups. As a result, this template offers customers a place to start that adheres with industry standards and may be adjusted to meet particular project requirements.
```
Comment (xmnlab) [inline README.md:6]:
```suggestion
Open Science Labs Scientific Python cookiecutter  template is based mainly on the PyOpenSci recommendations who is actively conducting research to determine the tools, libraries, best practices, and workflows utilized by significant scientific Python groups. As a result, this template offers to authors a place to start that adheres with industry standards and may be adjusted to meet particular project requirements.
```

**Review by xmnlab — CHANGES_REQUESTED** [review]
@Satarupa22-SD thanks for working on that!
I just added a suggestion here and please try to have lines with max 79 characters
thank you!

**Review by xmnlab — APPROVED** [review]
Looks good to me! Thanks for working on that!

**Pull request merged**